src/test/java/teammates/test/cases/ui/XlsxFileDownloadResultTest.java

```diff
@@ -0,0 +1,99 @@
+package teammates.test.cases.ui;
+
+import java.io.IOException;
+
+import org.apache.poi.xssf.usermodel.XSSFRow;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.testng.annotations.Test;
+
+import teammates.common.util.Const;
+import teammates.common.util.Sanitizer;
+import teammates.test.cases.BaseTestCase;
+import teammates.ui.controller.XlsxFileDownloadResult;
+
+public class XlsxFileDownloadResultTest extends BaseTestCase {
+
+    @Test
+    public void testEmptyContentXslFileDownloadResult() {
```

Comment:
Might be a typo here, I suspect the intention is to use "Xls" and not "Xsl".

Applies to other methods in this class as well.